src/constants/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import "./NavBar.css"
+
+function NavBar() {
+    return(
+        <nav>
+            <ul className="NavigationBar">
+                <li><NavLink className={({ isActive }) => isActive ? 'default-menu-link' : 'active-menu-link'}
+                             to="/">Home</NavLink></li>
+                <li><NavLink className={({ isActive }) => isActive ? 'default-menu-link' : 'active-menu-link'}
+                             to="new">Nieuwe post</NavLink></li>
```

Comment:
Ik geloof dat er een "/" voor de pagina naam moet staan. Dus <NavLink to ="/new"> en <NavLink to ="/blogposts">